static/app/components/onboardingWizard/deprecatedNewSidebar.tsx

```diff
@@ -0,0 +1,525 @@
+import {Fragment, useCallback, useEffect, useMemo, useState} from 'react';
```

Comment:
this file is just the copy as the `newSidebar` file as it is currently in the master branch